package.json

```diff
@@ -34,22 +35,28 @@
     "devDependencies": {
         "@testing-library/jest-dom": "^4.0.0",
         "@testing-library/react": "^8.0.7",
+        "@types/jest": "^24.0.18",
+        "@typescript-eslint/eslint-plugin": "^2.3.2",
+        "@typescript-eslint/parser": "^2.3.2",
         "babel-jest": "^24.8.0",
         "cross-env": "^5.2.0",
         "husky": "^2.4.1",
+        "jest": "^24.9.0",
         "lint-staged": "^8.1.5",
         "prettier": "^1.16.4",
         "proton-bundler": "github:ProtonMail/proton-bundler#semver:^1.0.0",
-        "proton-i18n": "github:ProtonMail/proton-i18n#semver:^1.4.0"
+        "proton-i18n": "github:ProtonMail/proton-i18n#semver:^1.4.0",
+        "proton-pack": "github:ProtonMail/proton-pack#semver:^2.0.0",
+        "ts-jest": "^24.1.0",
+        "typescript": "^3.6.3"
     },
     "dependencies": {
         "abortcontroller-polyfill": "^1.2.1",
+        "core-js": "^3.2.1",
```

Comment:
For some reason, even though core installs version 3+, it catches version 1.2.7 for core-js...
I think it would make sense to prevent using extraneous dependencies overall, if we can't be sure which version is being installed, or is there a good way to determine that?